README.md

```diff
@@ -7,64 +7,31 @@
 | --------------- | ---------------------------------------------------------------- |
 | License:        | Apache License, Version 2.0                                      |
 | Build:          | Gradle                                                           |
-| Documentation:  | https://docs.aerogear.org - Under construction                   |
+| End User Documentation:  | https://docs.aerogear.org - Under construction - [Github Link](./docs/modules/ROOT/pages/)|
+| Community Documentation: | https://aerogear.org       |
 | Issue tracker:  | https://issues.jboss.org/browse/AEROGEAR                         |
 | Mailing lists:  | [aerogear-dev](https://groups.google.com/forum/#!forum/aerogear) |
 
-## Documentation
-
-1. [End User Getting Started Guide](./docs/modules/getting-started/pages/getting-started.adoc)
-1. [Self Defence Checks](./docs/modules/getting-started/pages/auth-self-defence-checks.adoc)
-1. [Certificate Pinning](./docs/modules/getting-started/pages/certificate-pinning.adoc)
-1. [Troubleshooting](./docs/troubleshooting.adoc)
-1. [Service Contributor Guide](./docs/contributing-guide.adoc)
 
 ### List of SDKs
 
-AeroGear Services SDK consist of set of separate SDKs
+AeroGear Services consist of set of separate SDKs
+
+- Core - Core library used as base for other SDKs
+- Auth - Mobile Application Auth SDK (part of the core bundle)
+- Security - Mobile Application Self Defense Checks
```

Comment:
I know for now we have only self-defense check, but I think we should say it's a security module, not self-defense module.

Reply:
@danielpassos yes makes sense, I will update that in both prs, Android and iOS.

Reply:
@finp based on that comment should the file structure mentioned here: https://groups.google.com/forum/#!topic/aerogear/pdP8BFfxcTw be updated to be `using-auth-module.adoc`?

Reply:
@laurafitzgerald  let's stick with sdk for now